docs/BuildingLBANN.md

```diff
@@ -108,26 +122,31 @@ The following LLNL-maintained packages are optional.
     + Note that the environments provided here have a set of external
       packages and compilers that are installed on an LLNL LC CZ sytem.
       Please update these for your system environment.  Alternatively,
-      you can create baseline versions of the user-level spack configuation
+      you can create baseline versions of the user-level Spack configuation
       files and remove the externals and compilers from the spack.yaml
```

Comment:
This is not essential but it will be more readable if filenames (such as `spack.yaml`) and variables (such as following `LIBRARY_PATH` and so on) are back-quoted.

Reply:
Done.